<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>842</width>
    <height>858</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>831</width>
      <height>801</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="카카오톡피드백">
     <attribute name="title">
      <string>보고보고보고</string>
     </attribute>
     <widget class="QPlainTextEdit" name="feedbackField">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>781</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="saveFeedback">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>610</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>저장</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="kakaoNameField">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="isHome">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>결석</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendKakaoButton">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>610</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>**카톡전송**</string>
      </property>
     </widget>
     <widget class="QPushButton" name="saveExcel">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>610</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>엑셀 저장</string>
      </property>
     </widget>
     <widget class="QListView" name="classList">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>181</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QListView" name="studentList">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>10</y>
        <width>171</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="savePPT">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>610</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>ppt저장</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>진도</string>
      </property>
      <widget class="QPlainTextEdit" name="classCommentField">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>381</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>130</y>
        <width>401</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>특이사항</string>
      </property>
      <widget class="QPlainTextEdit" name="classSpecialField">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>381</width>
         <height>71</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QCheckBox" name="noKakaoCheck">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>카톡X</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addStudentButton">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>240</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>학생추가</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeStudentButton">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>240</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>학생삭제</string>
      </property>
     </widget>
     <widget class="QPushButton" name="removeClassButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>240</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>반삭제</string>
      </property>
     </widget>
     <widget class="QPushButton" name="addClassButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>240</y>
        <width>113</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>반추가</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
